### PR TITLE
Introducing vpc-go-sdk

### PR DESCRIPTION
**Pull request by astha-jain**

### CODE REVIEW

File: common/headers.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	HEADER_NAME_USER_AGENT = "User-Agent"
+
+	SDK_NAME = "go-sdk-template"
```
Comment (mkistler) [inline common/headers.go:11]: Need to fix this.

Comment (astha-jain) [reply common/headers.go]: Done 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,284 @@
...
+[bug report](<github-repo-url>/issues).
+Before that, please search for similar issues. It's possible that someone has already reported the problem.
+
+## Open source @ IBM
+Find more open source projects on the [IBM Github Page](http://ibm.github.io/)
+
+
+## Setting up VPC service
+
+A quick example to get you up and running with VPC Go SDK service in `US-South` region.
```
Comment (edabe) [inline README.md:128]: I would suggest redacting it as:

> A quick sample to get you up and running with the VPC Go SDK service in the Dallas (`us-south`) region.

Comment (astha-jain) [reply README.md]: Fixed.

**Pull request merged**